### DISCUSSION TITLE
Do I need hasura when using blitz?

### DISCUSSION THREAD

**Discussion by zwl1619** [blitz-js/blitz#1424] [Q&A] [answered]
1、Do I need [Hasura](https://github.com/hasura/graphql-engine) when using blitz?

2、Can I deploy a blitz application on my own server?

3、Support PWA？

**Comment by Skn0tt** [comment, marked as answer]
1. Nope, Hasura is not required. Blitz comes with [Prisma](https://www.prisma.io) as its database client by default.
2. Yes, you absolutely can. Just run `blitz build` and then `blitz start --production`, and it is running!
3. Yes, in the same way that Next.js supports PWAs.
